gradle.properties

```diff
@@ -1,5 +1,5 @@
 group = org.threadly
 version = 0.1.0-SNAPSHOT
-threadlyVersion = 3.8.0
-litesocketsVersion = 2.0.0
+threadlyVersion = 4.1.0
```

Comment:
It would, but its a direct dependency, so I like to keep it in the dependencies because of that.